### PR TITLE
Create a ReadMe file for Order

### PR DESCRIPTION
**Pull request by yymlion**

### CODE REVIEW

File: README_Order.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Order
+- This is a ReadMe file for Order.
```
Comment (jabezAtMlionCorp) [inline README_Order.md:2]: this is a readme file

**Review by jabezAtMlionCorp — APPROVED** [review]

**Review by hsinhenMlion — APPROVED** [review]
paynow $10

File: README_Order.md (hunk 1 above)
Comment (jabezAtMlionCorp) [inline README_Order.md:2]: this is another comment

**Review by jabezAtMlionCorp — APPROVED** [review]

**Review by kthiha-dev — COMMENTED** [review]
Better put a meaningful commit message. Such as `Created a readme file for order.`

**Review by hsinhenMlion — APPROVED** [review]

**Review by kthiha-dev — APPROVED** [review]

**Pull request merged**